examples/Actor/DemoActor/DemoActor.cs

```diff
@@ -67,7 +67,7 @@ public Task TestNoArgumentNoReturnType()
 
         public async Task RegisterReminder()
         {
-            await this.RegisterReminderAsync("TestReminder", null, TimeSpan.FromSeconds(5), TimeSpan.FromSeconds(5));
+            await this.RegisterReminderAsync("TestReminder", null, TimeSpan.FromSeconds(5), TimeSpan.FromSeconds(5), 2);
```

Comment:
We should make sure this doesn't change anything in the example.

Reply:
Created a new method